nvml/manifest.json

```diff
@@ -0,0 +1,48 @@
+{
+  "manifest_version": "2.0.0",
+  "app_uuid": "2c7a8b1e-9343-4b4a-bada-5091e37c4806",
+  "app_id": "nvml",
+  "display_on_public_website": true,
+  "tile": {
+    "overview": "README.md#Overview",
+    "configuration": "README.md#Setup",
+    "support": "README.md#Support",
+    "changelog": "CHANGELOG.md",
+    "description": "Support Nvidia GPU metrics in k8s",
+    "title": "Nvidia NVML",
+    "media": [],
+    "classifier_tags": [
+      "Category::Kubernetes",
+      "Category::OS & System",
+      "Supported OS::Linux",
+      "Supported OS::Windows",
+      "Supported OS::macOS"
+    ]
+  },
+  "author": {
+    "support_email": "help@datadoghq.com",
+    "homepage": "https://github.com/DataDog/integrations-core",
+    "sales_email": "help@datadoghq.com",
+    "name": "Datadog"
+  },
+  "oauth": {},
+  "assets": {
+    "integration": {
+      "source_type_name": "nvml",
+      "configuration": {
+        "spec": "assets/configuration/spec.yaml"
+      },
+      "events": {
+        "creates_events": false
+      },
+      "metrics": {
+        "prefix": "nvml.",
+        "check": "nvml.device_count",
+        "metadata_path": "metadata.csv"
+      },
+      "service_checks": {
+        "metadata_path": "assets/service_checks.json"
+      }
```

Comment:
If we have no service checks, I think we can remove that part and drop the `service_checks.json` file IIRC. Could you try and see if the validation works?